src/python/BUILD

```diff
@@ -81,7 +81,7 @@ cc_library(
     copts = COPTS_ADJUSTED,
     linkopts = LINKOPTS_ADJUSTED,
     alwayslink = 1,
-    data = ["//src/bindings/python:pyovms.so"],
+    data = ["//src/python/binding:pyovms.so"],
```

Comment:
I may not understand it well enough, but if we load Tensor class in runtime then don't we need this line only in the OVMS binary?